rn/Teacher/ios/Teacher/TeacherTabBarController.swift

```diff
@@ -57,7 +57,7 @@ class TeacherTabBarController: UITabBarController {
     }
 
     func toDoTab() -> UIViewController {
-        let todo = UINavigationController(rootViewController: TodoListViewController.create())
+        let todo = StyledNavigationController(rootViewController: TodoListViewController.create())
```

Comment:
It's strange that we only need to set this `StyledNavigationController` here. In the student app we use `HelmNavigationController`s in the tab bar controller and it seems to work, I wonder why.